### PR TITLE
Adds Acid pools

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request
2x3 structure, 200 tac points, heals sunder at a rate of 1 per 2 seconds when you rest on it (subject to change)

## Why It's Good For The Game
Hivelord no longer soft mandatory + more things to spend on

## Changelog
:cl:
add: Added acid pools, a sunder healing structure for xenos purchasable with tactical psychic points
/:cl:

### CODE REVIEW

**Comment by yyzsong** [comment]
XROS... I'D NEVER THOUGHT I'D SEE THE DAY

**Comment by TiviPlus** [comment]
requires tac points pr to compile btw

**Review by Kuro020 — APPROVED** [review]
I thought i'd never see the day tivi would stop being lazy

**Comment by Ragecrabbo** [comment]
Great addition in my opinion. 
Lack of any defenses of its own and requiring to be placed somewhat close to where the fighting is means its always at risk of being destroyed and unlike phero towers you cant just put it in a hard to reach spot and wall it up. One thing i'd like to see added to it is maybe a tiny healing buff for flavor and so xenos without meaningful amounts of armor or sunder get a benefit out of resting there, which in turn could better enable counterplay from smart marines using tarp/cas/offscreen weapons.
I hope to see this ingame either way though.

File: code/modules/mob/living/carbon/xenomorph/hive_upgrades.dm [hunk 1]
```diff
@@ -240,6 +240,28 @@ GLOBAL_LIST_INIT(tier_to_primo_upgrade, list(
 	flags_gamemode = ABILITY_NUCLEARWAR
 	building_type = /obj/structure/xeno/spawner
 
+/datum/hive_upgrade/building/acid_pool
+	name = "Larva Pool"
```
Comment (ivanmixo) [inline code/modules/mob/living/carbon/xenomorph/hive_upgrades.dm:244]: Larva pool

File: code/modules/xenomorph/xeno_structures.dm [hunk 2]
```diff
@@ -1637,3 +1637,32 @@ TUNNEL
 		X.alpha = initial(X.alpha)
 		balloon_alert(X, "Effect wears off")
 		to_chat(X, span_xenowarning("The effect of [src] wears off!"))
+
+/obj/structure/xeno/acid_pool
+	name = "acid pool"
+	desc = "A pool of weak viscous acid that solidifies quickly when removed from it's pool. Swimming is not reccomended due to the lack of lifeguard."
```
Comment (ivanmixo) [inline code/modules/xenomorph/xeno_structures.dm:1643]:
```suggestion
	desc = "A pool of weak viscous acid that solidifies quickly when removed from the pool. Swimming is not recommended due to the lack of a lifeguard."
```

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Comment by novaepee** [comment]
image

**Comment by lbnesquik** [comment]
2x3? That's a weird shape.

File: code/modules/mob/living/carbon/xenomorph/hive_upgrades.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/mob/living/carbon/xenomorph/hive_upgrades.dm:244]:
```suggestion
	name = "Acid Pool"
```

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**